script/Arcueid/Day 8/08_00_ARC08_4.txt

```diff
@@ -92,42 +101,55 @@ And while her back was turned, I spoke up,
 [sha:63f7d9149af669fbb937f931567a31457242b19b]{
 -- Page 237, Offset 20262.
 -- なんとなく、そんな質問を投げかけた。
-I throw out that question haphazardly.
+I lob him a vague question.
 }
 [sha:af0066be85716c3392bf8a6055affe8919bc2db5]{
 -- Page 238, Offset 20263. Mods: @k, @e.
 -- 「なんだい突然に。
-"Where did that come from all of a sudden?
+"What are you so curious about all of a sudden?
 }
 [sha:88cfc862302769db5dfcde1eb25dd404e0307d0c]{
 -- Page 238, Offset 20264. Glued. Mods: @x.
 -- 北口デパート並みにヤバめなスポットなんてそうそうねえぞ。……しいていうならアレか。<櫛塚|くしづか>の廃病院じゃねえか？　二年前まで<珍走団|ヤンキーども>の集会場になってたものの、今じゃ誰もよりつかなくなったっていう」
-There's hardly any other spot as bad as the northern department store... Well there is one if you ask me. It has to be the abandoned hospoital in Kushizuka right? It used to be where all those yankees gathered until two years ago, but nowadays it's become completely deserted."
+// Leading space for glue
+ There aren't really any spots that are as dodgy as the department store by the north gate...
+ If you pressed me on it, I guess there's one?
+ The abandoned hospital in Kushizuka fits the bill, right?
+ Some <biker gang|y a n k e e s   > used to be based out of there up until two years ago,
```

Comment:
I'd say just drop the ruby here tbqh. Doesn't really add anything besides a little bit of _Nihon flare_